lib/phlex/rails/checkbox_builder.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Phlex::Rails::CheckboxBuilder < Phlex::Rails::Decorator
+	def text(...) = @object.text(...)
+	def value(...) = @object.value(...)
+	def object(...) = @object.object(...)
+
+	output def label(...) = nil
+	output def checkbox(...) = nil
```

Comment:
What's the reason for defining outputs this way, as opposed to
```rb
output :label
output :checkbox
```
Is it better for the language server?

Reply:
It's for the language servers.